src/js/vic-v2/components/IntroductionPage.jsx

```diff
@@ -10,63 +10,143 @@ class IntroductionPage extends React.Component {
+      </div>
+    );
+
     return (
       <div className="schemaform-intro">
-        <FormTitle title="Apply for VIC"/>
+        <FormTitle title="Apply for a Veteran ID Card"/>
         <SaveInProgressIntro
           prefillEnabled={this.props.route.formConfig.prefillEnabled}
           pageList={this.props.route.pageList}
-          startText="Start the VIC application"
+          startText="Start the VIC Application"
+          resumeOnly
+          renderSignInMessage={this.renderSignInMessage}
           {...this.props.saveInProgressActions}
           {...this.props.saveInProgress}>
           Please complete the VIC form to apply for a card
         </SaveInProgressIntro>
+        <h4>Follow the steps below to apply for a Veteran ID Card.</h4>
         <div className="process schemaform-process schemaform-process-sip">
-          <h4>Follow the steps below to apply for VIC.</h4>
           <ol>
             <li className="process-step list-one">
               <div><h5>Prepare</h5></div>
-              <div><h6>To fill out this application, you’ll need information about the deceased Veteran, including their:</h6></div>
-              <ul>
-                <li>Social Security number or VA file number (required)</li>
-                <li>Date and place of birth (required)</li>
-                <li>Date and place of death (required)</li>
-                <li>Military status and history</li>
-              </ul>
-              <div><h6>You may need to upload:</h6></div>
+              <div><h6>When you apply, be sure to have these on hand:</h6></div>
               <ul>
-                <li>A copy of the deceased Veteran’s DD214 or other separation documents</li>
-                <li>A copy of the Veteran’s death certificate</li>
-                <li>Documentation for transportation costs (if you’re claiming costs for the transportation of the Veteran’s remains)</li>
+                <li>Your Social Security number</li>
+                <li>A digital copy of your DD214 or other military discharge papers that you can upload</li>
+                <li>A current color photo of yourself</li>
               </ul>
-              <p><strong>What if I need help filling out my application?</strong> An accredited representative with a Veterans Service Organization (VSO) can help you fill out your claim. <a href="/disability-benefits/apply/help/index.html">Find an accredited representative</a>.</p>
+              <div className="vic-intro-indent">
+                <div><h6>The photo has to follow all the standards listed below. It must:</h6></div>
+                <ul>
+                  <li>Show a full front view of your face and neck, (with no hat, head covering, or headphones covering or casting shadows on your hairline or face), **and**</li>
```

Comment:
Is this the latest content? I've seen another version but I'm not sure which is current: "Show a full front view of your face and neck, with full visibility of the face and no shadows, ***and***